apps/wallet-mobile/src/features/Notifications/common/notification-manager.ts

```diff
@@ -22,8 +19,8 @@ export const useNotificationManagerMaker = () => {
   return React.useMemo(
     () =>
       notificationManagerMaker({
-        eventsStorage: notificationStorage.join('events/'),
-        configStorage: notificationStorage.join(`settings/${walletId}/`),
+        eventsStorage,
```

Comment:
@michaeljscript question, the events are global not per wallet? if so when re-syncing the wallet should it be clear ?

Reply:
dunno if it should or shouldn't clear anything while resync, maybe since the user can clear them by themselves, let's leave as it is.